lib/engine/tile.rb

```diff
@@ -395,6 +395,14 @@ def city_town_edges
       ct_edges.values
     end
 
+    def city_town_edges_are_subset_of?(other_cte)
+      cte = city_town_edges
+      ALL_EDGES.any? do |rotation|
+        rotated = other_cte.map { |city| city.map { |edge| rotate(edge, rotation) } }
+        cte.all? { |city| rotated.any? { |other_city| (city - other_city).empty? } }
```

Comment:
what's city and other city? edges?

Reply:
Yes, an array of integers containing the edge numbers connected to that city.